lichess-bot.py

```diff
@@ -887,16 +898,25 @@ def pgn_game_record(li: lichess.Lichess, config: Configuration, game: model.Game
     return game_record.accept(pgn_writer)
 
 
-def get_game_file_path(config: Configuration, game: model.Game, force_single: bool = False) -> str:
+def get_game_file_path(config: Configuration,
+                       game_id: str,
+                       white_name: str,
+                       black_name: str,
+                       user_name: str,
+                       game_is_over: bool,
+                       *, force_single: bool = False) -> str:
```

Comment:
Why is `, *,` used?

Reply:
To force the use of `force_single=True` rather than just `True` in the last parameter. I was looking at these lines:
```python
game_path = get_game_file_path(config, game_id, white_name, black_name, user_name, game_is_over)
single_game_path = get_game_file_path(config, game_id, white_name, black_name, user_name, game_is_over, force_single=True)
```
Without the star, get `single_game_path` line could look like this:
```python
single_game_path = get_game_file_path(config, game_id, white_name, black_name, user_name, game_is_over, True)
```
with no obvious clue as to what `True` means. I had considered doing the same thing for the `game_is_over` parameter, but the function isn't called with a literal True or False. I could move the `*` before `game_is_over` for consistency.

Reply:
I don't think there is any reason to move it. The variable name `game_is_over` and `is_game_over(game)` shows what the parameter does.